### PR TITLE
Patch 1 - add `mojs-player ` repo initial files

### PR DESCRIPTION
**Pull request by legomushroom**

### CODE REVIEW

**Comment by jimaek** [comment]
No tags https://github.com/legomushroom/mojs-player/releases
The bot will fail

**Comment by legomushroom** [comment]
@jimaek made a release.

File: files/mojs-player/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author      = "Oleg Solomka"
+github      = "https://github.com/legomushroom/mojs-player"
+homepage    = "https://github.com/legomushroom/mojs-player"
+description = "Player controls for [mojs](mojs.io). Intended to help you to craft `mojs` animation sequences."
+mainfile    = "build/mojs-player.min.js"
```
Comment (jimaek) [inline files/mojs-player/info.ini]: Just `mojs-player.min.js` here

**Comment by legomushroom** [comment]
@jimaek thanks, changed

**Pull request merged**